src/OrchardCore.Cms.Web/Startup.cs

```diff
@@ -1,12 +1,42 @@
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging;
 using OrchardCore.Logging;
+using OrchardCore.OpenId.EntityFrameworkCore.Models;
 
 namespace OrchardCore.Cms.Web
 {
+
+    public class MyContext : DbContext
```

Comment:
Note: these changes will be reverted once this PR is ready. They are just here to show you how one can use the OpenID module as a management UI for an existing OpenIddict deployment (you basically just have to ignore the Orchard-specific model properties).